### DISCUSSION TITLE
Unraid Macos Docker Help

### DISCUSSION THREAD

**Discussion by dj13990** [RobertDWhite/WhiteMatterTech#14] [Q&A] [answered]
Hello I was following your guide on setting up a Macos docker.  Everything seemed to go good until I followed the Compose up part.  I get this error " ✘ macos Error                                                                                                     0.8s 
Error response from daemon: pull access denied for docker-osx-vnc, repository does not exist or may require 'docker login': denied: requested access to the resource is denied"

**Comment by RobertDWhite** [comment, marked as answer]
@dj13990 can you update the compose to use "docker-osx" instead of "docker-osx-vnc"? Try that, and let me know how it goes! I may need to update my post.
Thanks!

**Comment by dj13990** [comment]
That worked.  TY.  It also works if you use the vnc with “latest” at the end.

Thanks again for your help.

-Doug

**Reply by RobertDWhite** [reply]
Awesome! Glad to help!
